### ISSUE TITLE
[AL] Ship new model from the paper

### ISSUE THREAD

**Issue by JulienPeloton** [astrolabsoftware/fink-science#122]
The Active Learning paper introduces a new model. The training set has changed, and the features have been updated. 

Action items:
- [x] default model should be changed
- [x] introduce a convention parameter as the order probabilities returned by the binary classifier has changed (-1 if `[pIa, pNIa]`, +1 if `[pNIa, pIa]`)
- [ ] update the `chi2` feature by normalising the integral of parameters
- [x] force probabilities to be zero if one of the 2 bands is missing (i.e. allows only candidates with 2 valid bands to return a non-zero probability).

**Comment by JulienPeloton** [comment]
chi2 does not need normalisation after all.
